### ISSUE TITLE
2.2.0 release

### ISSUE THREAD

**Issue by blairconrad** [FakeItEasy/FakeItEasy#772]
Can be labelled **ready** when all other issues on this milestone are closed.

- [x] run code analysis in VS in *Release* mode and address violations (send a regular PR which must be merged before continuing)
- [x] if necessary, change `VERSION_SUFFIX` on the [CI Server](http://teamcity.codebetter.com/admin/editBuildParams.html?id=buildType:bt929)
      to appropriate "-beta123" or "" (for non-betas) value and initiate a build
- [x] check build
- edit draft release in [GitHub UI](https://github.com/FakeItEasy/FakeItEasy/releases):
    - [x] ensure completeness of release notes, including non-owner contributors, if any (move release notes forward from any pre-releases to the current release)
    - [x] attach main nupkg and/or analyzer nupkg, whichever have content to release
    - [x] publish the release
- [x] push nupkg(s) to NuGet
- [x] de-list pre-release or superseded buggy NuGet packages if present
- [x] update website with contributors list (if in place)
- [x] tweet, mentioning contributors and post link as comment here for easy retweeting ;-)
- [x] post tweet in [Gitter](https://gitter.im/FakeItEasy/FakeItEasy)
- [x] post a link to the GitHub Release in each issue in this milestone, with thanks to contributors
- [x] run `rake next_version[new_version]` to
    - create a pull request that changes the version in CommonAssemblyInfo.cs to the expected version (of form _xx.yy.zz_)
    - create a new draft GitHub Release
    - create a new milestone for the next release
    - create a new issue (like this one) for the next release, adding it to the new milestone
- [x] close this milestone

**Comment by adamralph** [comment]
Ready to go!

**Comment by adamralph** [comment]
I can probably take this on tomorrow.

**Comment by thomaslevesque** [comment]
> I can probably take this on tomorrow.

Let me know if you need some help. It's France's National Day tomorrow, so I won't be at work :wink:

**Comment by adamralph** [comment]
@thomaslevesque oh, by all means, if you're free tomorrow please feel free to do the release. I was going to fit it  in between work tasks, so my day would be easier without it.

**Comment by thomaslevesque** [comment]
No problem, I'll take care of it

**Comment by adamralph** [comment]
https://twitter.com/fakeiteasyfx/status/753581763688034304

**Comment by thomaslevesque** [comment]
> https://twitter.com/fakeiteasyfx/status/753581763688034304

Ah, forgot to post the link... Thanks :wink:

**Comment by blairconrad** [comment]
I forgot last time too. I wonder if we shouldn't split that line.

Thanks for your work, @thomaslevesque!
